packages/blade/src/components/Box/BaseBox/baseBoxStyles.ts

```diff
@@ -221,6 +221,7 @@ const getAllProps = (
       breakpoint,
     ),
     borderStyle: hasBorder ? 'solid' : undefined,
+    cursor: getResponsiveValue(props.cursor, breakpoint),
```

Comment:
:p why does cursor needs to be a responsive prop?

Reply:
You never know. What if they are turning a clickable item into non-clickable in mobile 😆 

Anyway doing it for consistency because everything in Box is responsive prop